OpenOversight/app/main/views.py

```diff
@@ -1681,9 +1659,7 @@ def redirect_download_dept_descriptions_csv(department_id: int):
     )
 
 
-@main.route(
-    "/download/departments/<int:department_id>/descriptions", methods=[HTTPMethod.GET]
-)
+@main.get("/download/departments/<int:department_id>/descriptions")
```

Comment:
Maybe out of scope for this pull request but the CSV for this route is named Notes instead of Descriptions